view/frontend/templates/breadcheckout/view.phtml

```diff
@@ -195,11 +195,16 @@
                 breadConfig.actAsLabel = false;
             }
 
-            if(cartSizeFinancing.enabled){
-                var cartSizeFinancingId = cartSizeFinancing.id;
-                var cartSizeThreshold = cartSizeFinancing.threshold;
+            if(isFinancing.enabled && isFinancing.mode === 1){
+                var cartSizeFinancingId = isFinancing.id;
+                var cartSizeThreshold = isFinancing.threshold;
                 var itemsPriceSum = items.reduce(function(sum, item) { return sum + item.price * item.quantity; }, 0) / 100;
                 breadConfig.financingProgramId = (itemsPriceSum >= cartSizeThreshold) ? cartSizeFinancingId : 'null';
+            } else if (isFinancing.enabled && isFinancing.mode === 2){
+                var productSku = "<?= $block->getProduct()->getSku(); ?>";
+                if(isFinancing.sku_limit.includes(productSku) === true){
```

Comment:
don't think the === true is needed, but if this was on purpose for readability/clarity then lemme know

Reply:
just for clarity, it's not a problem as they are and will be of same type, so would leave it as is.